src/api/habits/adapters/json.ts

```diff
@@ -26,7 +30,75 @@ const jsonHabitServiceFactory = (): HabitService => {
         }))
       );
 
-  return { addHabit, getHabits };
+  const addEntry = (
+    goals: GoalWithHabitHistory[],
+    habitId: string,
+    date: Date = new Date(),
+    quantity: number = 1
```

Comment:
I recommend adding these sorts of things as an object, easily extensible, do not suffer from long, ordered parameters either, which are harder to known the meaning of without looking up the original function definition

Reply:
I've done this for addEntry(), but is it worth doing it for removeEntry() too, which has 2 params (at the moment)?

Reply:
It's worth doing for parameters which are fundamentally "data". For functions which are not (delete is an example, since it only really requires one identifier), you don't need to unless you'd like the call signature to look consistent